settings.gradle

```diff
@@ -1,4 +1,6 @@
 include 'core'
 include 'tools'
 include 'examples'
-include 'wallettemplate'
+if (JavaVersion.current().isJava11Compatible()) {
```

Comment:
Unfortunately Gradle 3.4.1 (which is the version that is in Ubuntu 18.04) does not have `.isJava11Compatible()`. It would have `.isJava10Compatible()` though, in case the javafxplugin is compatible with Java 10, too.

Reply:
Even Gradle 4.4.1 (the current version on Debian testing and Ubuntu 19.04) doesn't have `.isJava11Compatible()`.

Reply:
> in case the javafxplugin is compatible with Java 10, too.

Java 11 is the first version that doesn't include JavaFX by default. It's also the long-term release so it makes some sense that the plugin requires 11+.

Reply:
I agree. I'm just looking for workarounds. Another workaround may be using `getMajorVersion()` and do some string foo.

Reply:
Or maybe a construct like `isAtLeastGradle4.7() && isJava11Compatible()` would guard us from the exception? This would of course make the wallet-template build require Gradle 4.7.

Reply:
> workaround may be using `.getMajorVersion()` and do some string foo.

> a construct like `isAtLeastGradle4.7() && isJava11Compatible()`

This corresponds to "option 2" I suggested here: https://github.com/bitcoinj/bitcoinj/pull/1715#issuecomment-467086167

> This would of course make the wallet-template build require Gradle 4.7.

It looks like the OpenJFX plugin requires Gradle 4.10 or later.